NtupleProducer/python/HiggsTauTauProducer_106X.py

```diff
@@ -414,10 +414,18 @@
     jecLevels = [ 'L1FastJet', 'L2Relative', 'L3Absolute', 'L2L3Residual' ]
 
 
+process.bregJets = cms.EDProducer("bRegressionProducer",
+                                              JetTag = cms.InputTag("slimmedJets"),
+                                              rhoFixedGridCollection = cms.InputTag("fixedGridRhoAll"),
+                                              bRegressionWeightfile = cms.FileInPath("LLRHiggsTauTau/NtupleProducer/data/DNNmodels/breg_training_2018_JECv8.pb"),
```

Comment:
The weights are year-dependent, so the path should not be hard-coded.

Reply:
this needs to be confirmed

Reply:
adapted according to the twiki's information (not yet fully certain it is up to date)